### ISSUE TITLE
iOS: Title tap zone on Timeline should be labeled as actionable for accessibility 

### ISSUE THREAD

**Issue by vincode-io** [Ranchero-Software/NetNewsWire#1583]
When viewing a list of articles in a feed, the name of the feed displayed at the top of the screen should be indicated to VoiceOver users as being an actionable UI element.

Currently, when VoiceOver focus is on this element it speaks “FeedName Heading”.

VoiceOver users would not know from this that they can double-tap on this UI element to access the feed's info screen.

My suggestion would be to indicate that this is a button.
